ui/src/androidTest/java/com/xmartlabs/bigbang/ui/recyclerview/common/ListActivity.java

```diff
@@ -0,0 +1,27 @@
+package com.xmartlabs.bigbang.ui.recyclerview.common;
+
+import android.app.Activity;
+import android.os.Bundle;
+import android.support.v7.widget.LinearLayoutManager;
+import android.support.v7.widget.RecyclerView;
+
+public abstract class ListActivity<T extends RecyclerView.Adapter> extends Activity {
```

Comment:
What do you think about requiring a `RecyclerView` to be provided by the concrete class instead of doing all the setup here with information we don't yet have?